assets/chezmoi.io/docs/user-guide/tools/diff.md

```diff
@@ -18,8 +18,8 @@ specify:
 !!! hint
 
     If you generate your config file from a config file template, then you'll
-    need to escape the `{{` and `}}` in your config file template so that they
-    appear in your generated config file.
+    need to escape the `{{` and `}}` as `{{"{{"}}` and `{{"}}"}}`. That way
+    your generated config file contains the `{{` and `}}` you expect.
```

Comment:
Can you remove the `the` in this fragment (both files): `` the `{{` ``.?

If I look at this with my documentation editor hat on, I would probably rephrase the whole thing to

```markdown
If you generate your config file from a template, then you need to escape `{{` and `}}`
as `{{ "{{" }}` and `{{ "}}" }}` in your template so that your generated config file
contains `{{` and `}}` characters.
```

It *may* also be worth pointing people to the [delimiters template directive](https://www.chezmoi.io/reference/templates/directives/#delimiters) (`[delimiters template directives](reference/templates/directives/#delimiters)`)  as this can make the question of `{{` and `}}` irrelevant.

This comment applies to both files you have modified.

Reply:
Sorry for not coming back to this, I was traveling.

One thing I learned much more recently than I would like to admit is the GitHub suggestion feature — the left-most icon in the toolbar above. It looks like this:

```suggestion
    your generated config file contains `{{` and `}}` you expect.
```

I use this a lot in [another repository](https://github.com/google/comprehensive-rust) to apply all sorts of small and large fixups to PRs. Often, contributors don't really care about the exact wording, so as a maintainer it's much easier if I fix things and then squash-merge the branch on the spot.